messages/welcome.json

```diff
@@ -3,14 +3,14 @@
     {
       "header": "Enjoying rotki?",
       "icon": "https://raw.githubusercontent.com/rotki/data/develop/messages/icon.svg",
-      "text": "rotki is part of the GIV-a-Palooza QF Round at Giveth",
+      "text": "rotki is part of the Octant Epoch 5",
       "action": {
         "text": "Support us here",
-        "url": "https://giveth.io/project/rotki"
+        "url": "https://octant.app/home" 
```

Comment:
you don't need to delete an old one. Just add a new one. Timestamps alone should decide what shows.